src/routes/embedded/account/backup-shares/account-backup-shares.view.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEmbedded } from "~utils/embedded/embedded.hooks";
+import { DevFigmaScreen } from "~components/dev/figma-screen/figma-screen.component";
+
+import screenSrc from "url:/assets-beta/figma-screens/backup-options.view.png";
+
+export function AccountBackupSharesEmbeddedView() {
+  const { wallets, promptToBackUp, generateRecoveryAndDownload } =
+    useEmbedded();
+  const walletAddress = wallets[0].address;
+
+  // TODO: What if the user already has more than 3 backup shares?
```

Comment:
@Jonathansoufer I'll bring this up with design tomorrow. I'll leave the TODOs in place but I'm collecting all these in a document.